torch/__init__.py

```diff
@@ -609,8 +609,14 @@ def manager_path():
     # PR #43339 for details.
     from torch._C._VariableFunctions import *  # type: ignore[misc] # noqa: F403
 
+# Ops not to be exposed in `torch` namespace,
+# mostly helper ops.
+PRIVATE_OPS = (
+    'unique_dim',
+)
+
```

Comment:
I'm trying to understand the issue here. We don't want to expose torch.unique_dim. However, we want torch._C._VariableFunctions.unique_dim to work. Therefore we need to enable Python bindings for "unique_dim" and delete it here.

The reason why we can't rename "unique_dim" to "_unique_dim" is for BC reasons. Does that sound right?

Reply:
Yes. That's about it